src/dealWizard/components/proposalLeadSection/proposalLeadSection.html

```diff
@@ -4,24 +4,41 @@
       <h2 class="heading heading3 title">Proposal Lead</h2>
       <p class="subtext">
         Provide the Ethereum address for the proposal lead who will gain admin rights of the deal.
-        <br />
-        <br />
-        Proposal leads will be responsible for editing the deal during negotiation, initiating the funding phase once a deal is approved, and executing the token swap. You don’t need to be part of a DAO to be the lead proposal.
+        <br/>
```

Comment:
Ii think this should be done with a class + margin or padding, rather then brs

Reply:
I say those while editing (were there before) and I thought about the same thing. I added design system spacing classes and used those instead